### PR TITLE
Made filenames from stack traces in result view clickable.

### PR DESCRIPTION
**Pull request by pbihler**
(Had to be done in two steps using jQuery due to atom's Content Security Policy)

### CODE REVIEW

File: lib/clickable-paths.coffee [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  atom.workspace.open(filename)
+  .then ->
+
+    return unless row?
+    col ?= 0
+
+    editor = atom.workspace.getActiveTextEditor()
+    return unless editor?
+
+    position = new Point(row, col)
```
Comment (nguyenchr) [inline lib/clickable-paths.coffee]: looks like there is an off by one error, I changed this line locally to `position = new Point(--row, --col)` and it seems to land on the correct line

**Comment by nguyenchr** [comment]
Code looks great, thanks for this PR, just one minor comment about the `Point` coordinates

**Comment by pbihler** [comment]
Mixed the coordinates

**Comment by nguyenchr** [comment]
nice one :+1:

**Pull request merged**